tap_pendo/streams.py

```diff
@@ -345,10 +345,10 @@ def load_metadata(self):
 
         if self.replication_key:
             mdata = metadata.write(mdata, (), 'valid-replication-keys',
-                                   [self.replication_key])
+                                   [humps.decamelize(self.replication_key)])
```

Comment:
Please add a comment, why you used humps.decamelize

Reply:
Done

Reply:
For a certain stream like "features, track_types, pages, guides" the replication key in schema is "last_updated_at" and in class variable of stream it is "lastUpdatedAt" so rather than updating the replication key value in the class variable used "humps.decamelize" for backward compatibility as for previous syncs the value in the bookmark will contain "lastUpdatedAt"